src/Servers/Kestrel/Core/src/Internal/Http/Http1Connection.cs

```diff
@@ -642,6 +644,7 @@ protected override void BeginRequestProcessing()
     {
         // Reset the features and timeout.
         Reset();
+        _requestCount++;
```

Comment:
Request count is more accurate here. Reset is called from `BeginRequestProcessing` and also from `HttpProtocol.Initialize` so is 1 higher than it should be.

Some tests are broken that need to react to this.